### PR TITLE
Set primary color and fontsize in theme.js

### PR DESCRIPTION
**Pull request by alvyjudy**
primary color was set to the orange and base font size was set to 16px in theme.js

### CODE REVIEW

File: client/src/themes/theme.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { createMuiTheme } from "@material-ui/core";
 export const theme = createMuiTheme({
   typography: {
     fontFamily: '"Roboto"',
-    fontSize: 12,
+    fontSize: 16,
```
Comment (sina-jamshidi) [inline client/src/themes/theme.js]: I'll approve for now but isn't 16 kind of big? lol

**Review by sina-jamshidi — APPROVED** [review]
you're good to merge!

File: client/src/themes/theme.js (hunk 1 above)
Comment (alvyjudy) [reply client/src/themes/theme.js]: on second thought, it might actually be too big... I'll revert it to 12 😅

**Comment by alvyjudy** [comment]
Ticket number #29  (Forgot to link)

**Pull request merged**